src/services/storage.rs

```diff
@@ -50,7 +50,9 @@ impl StorageService {
             }
         };
         #[cfg(feature = "web_sys")]
-        let storage = storage.unwrap().unwrap();
+        let storage = storage
+            .and_then(|storage| storage.ok_or(JsValue::NULL))
+            .expect("failed to aquire storage");
```

Comment:
Apparently if the user blocks the website from using the storage API, this can fail.
I thought it would be best if `new` returns a `Result`/ `Option` in this case, but `stdweb` doesn't handle this case.

Reply:
I'd be more happy with proper`web-sys` support so yeah `Result` sounds great

Reply:
Should I try to modify the `stdweb` part with a `js!` and `new Storage` and a `catch` instead of using `local_storage()`/`session_storage()`?

Reply:
> Should I try to modify the `stdweb` part with a `js!` and `new Storage` and a `catch` instead of using `local_storage()`/`session_storage()`?

I did that now.